source/js/buyers-guide/search.js

```diff
@@ -0,0 +1,107 @@
+const ALL_PRODUCTS = document.querySelectorAll(`figure.product-box`);
+
+const filters = [
+  `company`,
+  `name`,
+  `blurb`,
+  `worst-case`,
+  // `categories`
```

Comment:
is this to be covered as follow-up?

Reply:
It wasn't clear if we really wanted categories, so I left the code in for now but will probably take it out when doing the final styling/responsive positioning. It doesn't really add anything meaningful.